site/content/en/docs/Reference/fleet.md

```diff
@@ -226,6 +238,13 @@ The `spec` field is the actual `Fleet` specification and it is composed as follo
   - `Fleet's Scheduling Strategy`: The GameServers associated with the GameServerSet are sorted based on either `Packed` or `Distributed` strategy.
       - `Packed`: Agones maximizes resource utilization by trying to populate nodes that are already in use before allocating GameServers to other nodes.
       - `Distributed`: Agones employs this strategy to spread out GameServer allocations, ensuring an even distribution of GameServers across the available nodes.
+- `priorities` (Alpha, requires `CountsAndLists` feature flag): Defines which gameservers in the Fleet are most important to keep around - impacts scale down logic.
```

Comment:
Tweaking some formatting and docs, but this is exactly what we need here and in the other .md files in this PR 👍🏻 

```suggestion
- `priorities`: (Alpha, requires `CountsAndLists` feature flag): Defines which gameservers in the Fleet are most important to keep around - impacts scale down logic.
```